### ISSUE TITLE
Silent notification displayed in Android

### ISSUE THREAD

**Issue by tapz** [OneSignal/react-native-onesignal#285]
In Android, when the app is active a push notification with contents, dat, and android_sound is not displayed, but when the push notification does not have any of those fields (= it should be silent regardless if the app is active or not) the notification is displayed in the Notification Shade. I have set inFocusDisplaying to None.

OneSignal.inFocusDisplaying(0);

"react-native-onesignal": "3.0.4"
React Native: 0.46.2
Device: Moto G 4th gen, Android 7.0

**Comment by tapz** [comment]
Is this a bug in react-native-onesignal or in the One Signal Android SDK? Easily reproducible with curl:

```
curl --include \
     --request POST \
     --header "Content-Type: application/json" \
     --header "Authorization: Basic 7b8ayb890yaep89vby8adpb89adyb89HIOH99IHiohhihihih" \
     --data-binary "{\"content_available\": \"true\", \"app_id\": \"000000000-0000-0000-0000-000000000000\", \"priority\": \"10\", \"android_sound\": \"nil\", \"android_visibility\": \"-1\", \"android_group\": \"mygroup\", \"filters\": [{\"field\": \"tag\", \"key\": \"abc\", \"relation\": \"=\", \"value\": \"true\"}, {\"field\": \"tag\", \"key\": \"efg\", \"relation\": \"=\", \"value\": \"true\"}]}" \
     https://onesignal.com/api/v1/notifications
```

**Comment by Alamgir** [comment]
@tapz `inFocusDisplaying` only refers to the notification behavior when the application is in the foreground. I'm unable to produce a case where `inFocusDisplaying` is set to `NONE` and the notification appears in the shade while the app is foregrounded. I used the same payload parameters as your curl call

Are you trying to prevent the notification from appearing in the shade even when it's in the background?

**Comment by Alamgir** [comment]
@tapz upon further investigation, it looks like this issue should be resolved as it was addressed in the 3.6.2 release - since the react project takes the latest 3.x OneSignal build, these fixes should be already incorporated.
